onnx_neural_compressor/algorithms/weight_only/gptq.py

```diff
@@ -380,6 +380,7 @@ def gptq_quantize(
     if return_modelproto:
         return model.model
     else:
+        model.save(model.model_path + "_quant.onnx")
```

Comment:
Do we need to use "onnx.save_model" if the model is large?

Reply:
we call save function of ONNXModel and it will use onnx.save_model if the model is a large model